.github/workflows/platform.dependencies.yml

```diff
@@ -15,10 +15,14 @@ env:
   defaultLocation: 'WestEurope'
   resourceGroupName: 'validation-rg'
   resourceGroupNameArtifacts: 'artifacts-rg'
+  platform_resourceGroupName: 'platform-core-rg'
```

Comment:
**NOTE**: This will be renamed to align with others once other comments are addressed

Reply:
So this will be what exactly? 'platform-rg'? or 'management-rg'?

Reply:
The env. variable name will be `resourceGroupNamePlatform`. The resource group name will stay the same